### PR TITLE
feature: Add messenger page

### PR DESCRIPTION
**Pull request by JimTheCat**
Create basic construction of messages page

### CODE REVIEW

**Comment by JimTheCat** [comment]
Task linked: [CU-8696pcyn4 Create messages page](https://app.clickup.com/t/8696pcyn4)

**Comment by JimTheCat** [comment]
This is how it looks like:
image

**Review by KinTrae — CHANGES_REQUESTED** [review]

File: frontend/src/Pages/Messenger/Messenger.tsx [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+    };
+
+    users.forEach((user) => {
+      setInterval(() => {
+        user.lastActiveAt = new Date();
+        if (user.status !== "online") {
+          user.status = "online";
+          updateUserStatusInApp(user);
+        }
+      }, getCryptoRandom() * 300000 + 60000); // Aktywność co 1-6 minut
```
Comment (KinTrae) [inline frontend/src/Pages/Messenger/Messenger.tsx:197]: i still see here polish comments

File: frontend/src/Pages/Messenger/Messenger.tsx [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+    );
+  }
+
+  const simulateUserStatusChanges = (users: DummyUserType[]) => {
+    const crypto = window.crypto;
+
+    const getCryptoRandom = () => {
+      const array = new Uint32Array(1);
+      crypto.getRandomValues(array);
+      return array[0] / (0xFFFFFFFF + 1); // Normalizacja do zakresu 0–1
```
Comment (KinTrae) [inline frontend/src/Pages/Messenger/Messenger.tsx:168]: and here

**Review by KinTrae — CHANGES_REQUESTED** [review]

**Review by KinTrae — APPROVED** [review]

**Pull request merged**